keyboards/framework/laptop_16/readme.md

```diff
@@ -0,0 +1,36 @@
+# Framework Laptop 16 Keyboard
+
+Keyboard input modules for the Framework Laptop 16.
+
+* Keyboard Maintainer: [Daniel Schaefer](https://github.com/JohnAZoidberg)
+* Hardware Supported: Framework Laptop 16 Keyboard. Soon: Numpad and Macropad
+* Hardware Availability: Available at https://frame.work/marketplace
+
+
+## Vendor specific
+On the Framework 16, KC_MSEL launches the Framework website
+
+## Variants
+
+| Name           | Size     | Backlight  | Keys |
+| -------------- | -------- | ---------- | ---- |
+| ansi/white     | Keyboard | White      |   78 |
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb framework/laptop_16/ansi/white -km default
+
+Flashing example for the ANSI keyboard:
+
+    qmk compile -kb framework/laptop_16/ansi/white -km default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **On Framework Laptop 16 Keyboard**: Hold down left ALT and right ALT while installing the module
+* **On Raspberry Pi Pico**: Hold down bootsel button when plugging in
```

Comment:
Is this relevant to the actual hardware? Pi Pico is a development board.

Reply:
It's not. I assume it was added in as some sort of placeholder that the original, original PR maker put while building the firmware. I'll remove that.

Reply:
Removed in d314a86358282862ce2cb96e9ca43709737587c4